medusa/server/core.py

```diff
@@ -202,8 +202,11 @@ def run(self):
             protocol = 'http'
             self.server = HTTPServer(self.app)
 
-        logger.log('Starting Medusa on {scheme}://{host}:{port}/'.format
-                   (scheme=protocol, host=self.options['host'], port=self.options['port']))
+        logger.log('Starting Medusa on {scheme}://{host}:{port}{web_root}'.format
```

Comment:
Does this not need `/` before and after the `{web_root}` or does `web_root` include a starting and ending one?